client/src/components/Mentions.jsx

```diff
@@ -4,28 +4,40 @@ import styles from '../styles/Mentions.css';
 class Mentions extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      expanded: false,
+    };
+  }
+
+  handleSeeMoreClick() {
+    const newExpanded = !this.state.expanded;
+    this.setState({
+      expanded: newExpanded,
+    });
   }
 
   render() {
-    const state = this.props.getState();
-    const { reviews } = state;
+    const mentionPhrases = ['lorem', 'ipsum', 'is', 'simply', 'dummy', 'text', 'of', 'the', 'printing', 'and', 'typesetting', 'industry', 'lorem', 'ipsum', 'is', 'simply', 'dummy', 'text', 'of', 'the', 'printing', 'and', 'typesetting', 'industry'];
+
+    let mentions = mentionPhrases.map((phrase, index) => {
+      return <div key={index} className={styles.mentionItem}>{phrase}</div>;
```

Comment:
does ESLint complain that "Do not use array index in keys"? i keep getting that

Reply:
I don't see problems on this line, but I'm using the AirBnB eslint config, not HR